src/com/serotonin/mango/web/dwr/SystemSettingsDwr.java

```diff
@@ -394,7 +397,12 @@ public DwrResponseI18n saveMiscSettings(int uiPerformance, String dataPointRtVal
 		else {
 			response.addContextualMessage(SystemSettingsDAO.WEB_RESOURCE_UPLOADS_PATH, "systemsettings.webresource.uploads.path.wrong", File.separator);
 		}
-		SystemSettingsService systemSettingsService = new SystemSettingsService();
+		if(pointExtendedNameLengthInReportsLimit < 0) {
+			response.addContextualMessage(SystemSettingsDAO.DATA_POINT_EXTENDED_NAME_LENGTH_IN_REPORTS_LIMIT, "validate.invalidValue");
+		}
+		else{
```

Comment:
`} else { ` 

The above also needs to be improved.

Use:
if (condition) {
...
} else {
...
}